lib/manager/components/transform/FeedTransformation.js

```diff
@@ -94,9 +95,14 @@ export default class FeedTransformation extends Component<Props, {errors: Array<
 
   render () {
     const {feedSource, index, transformation} = this.props
-    const tables = [...getGtfsSpec()].filter(t => !t.datatools)
-    if (isModuleEnabled('gtfsplus')) tables.push(...getGtfsPlusSpec())
     const transformationType = transformation['@type']
+    const tables = [...getGtfsSpec()]
+    // Do not offer to transform datatools-internal tables
+      .filter(t => !t.datatools)
+      // Non-csv flex tables can only be replaced from another version. Otherwise, filter them out
+      .filter(t => transformationType === FEED_TRANSFORMATION_TYPES.REPLACE_FILE_FROM_VERSION ? true : !t.flex)
```

Comment:
This could just be an or.

Reply:
Good catch thank you! Given that this used to be a 5 line if/else block during development I'm so happy to see it coming down.